File: P1-Linguagem_de_Programação_I/Roteiros de Aula - Python/roteiro-de-aula04-exemplo3.py
matriz_A = []

print("--- Digite 10 nomes ---")
for i in range(10):
    nome = input(f"Digite o {i+1}º nome: ")
    matriz_A.append(nome)

print("\n--- Nomes Inseridos ---")
for nome in matriz_A:
    print(nome)

File: P1-Linguagem_de_Programação_I/Roteiros de Aula - Java/roteiro-de-aula04-exemplo3.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        // Usamos 'String[]' para arrays de Strings de tamanho fixo
        String[] matrizA = new String[10];

        System.out.println("--- Digite 10 nomes ---");
        for (int i = 0; i < 10; i++) {
            System.out.print("Digite o " + (i + 1) + "o nome: ");
            matrizA[i] = scanner.nextLine();
        }

        // Apresentar os nomes
        System.out.println("\n--- Nomes Inseridos ---");
        for (String nome : matrizA) {
            System.out.println(nome);
        }
    }
}